specs/models/ModelDefinitions.json

```diff
@@ -75,6 +75,12 @@
                 "description": {
                   "type": "string"
                 },
+                "namespace": {
```

Comment:
This doesn't seem like the right location - these are definition-level properties right, not properties-level properties?